examples/build.gradle

```diff
@@ -67,6 +70,8 @@ task benchmark(type: JavaExec) {
                         environment("OMP_NUM_THREADS", "1")
                         environment("TF_NUM_INTEROP_THREADS", "1")
                         environment("TF_NUM_INTRAOP_THREADS", "1")
+                    } else if (System.properties["ai.djl.default_engine"] == "DLR") {
+                        environment("TVM_NUM_THREADS", Runtime.getRuntime().availableProcessors())
```

Comment:
This doesn't do anything right now?

Reply:
If we assume DLR models are lightweight, should we give it 2X of cores?

Reply:
I will collect the benchmark data